.travis.yml

```diff
@@ -21,7 +21,9 @@ before_install:
     - source ./.travis/heroku-helper.sh
     - sudo pip install awscli
     - curl https://cli-assets.heroku.com/install-ubuntu.sh | sh
-    - sudo apt-get purge -y postgresql && sudo apt-get install postgresql-10 postgresql-client-10
+    - wget -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O - | sudo apt-key add -
+    - sudo apt-get update && sudo apt-get install postgresql postgresql-contrib
+    #- sudo apt-get purge -y postgresql && sudo apt-get install postgresql-10 postgresql-client-10
```

Comment:
probably a good idea to remove this commented section unless we plan on going back to it later